app/api/ada/lib/utils.js

```diff
@@ -53,3 +61,40 @@ const _getTxCondition = (state: string): AdaTransactionCondition => {
   if (state === 'Pending') return 'CPtxApplying';
   return 'CPtxWontApply';
 };
+
+export function decodeRustTx(resp: SpendResponse): CryptoTransaction {
```

Comment:
if we only use `cbor_encoded_tx` I would make that the input of the function

Reply:
I wanted to do it too, but then input type would be `Array<number>` which looks kinda weird 😟 I will check if I can maybe easily add a new type alias, like `RawRustTxBody`, or something like that 🤔

Reply:
Added new type alias `RustRawTxBody`: https://github.com/Emurgo/yoroi-frontend/pull/160/commits/0767e6d8b5c73452c9715d27b4ceebc5ae8c076b

Reply:
oh good idea